README.md

```diff
@@ -1,2 +1,31 @@
-# third-party-test-harness
+# Third-Party-Test-Harness
+
+## Description:
 A small Java web application that is used primarily for the use of karate testing and interacting with CHS services. The testing with this harness focuses on obtaining the correct scopes and permissions that are requested via this web application for use of Companies House Enabled API's
+This is an example application of a third party app gaining an Oauth token and using it for access and displaying,
+the correct scopes for access to API's. 
+
+### Requirements:
+- [Java](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+- [MongoDB](https://www.mongodb.com)
+
+### Usage:
+* In application.properties file changes need to be made to environment variables in order to get the application to work
+* Entrypoint is [HOME](http://localhost:8090/login)
```

Comment:
If we want a link here I would use the same text as the url

Reply:
Changed the type from link so visible